<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZOperationsForm</class>
 <widget class="QWidget" name="ZOperationsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>434</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Добавление (изменение) данных по операции</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../plan_fakt.qrc">
    <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="requiredBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="nameText">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Дата операции:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDateEdit" name="dateEdit">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>225</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="nameText_4">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Тип:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QComboBox" name="cboType">
        <item>
         <property name="text">
          <string>Поступление</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Выплата</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Перемещение</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="nameText_2">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Юр. лицо:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QComboBox" name="cboUrPerson"/>
      </item>
      <item row="2" column="3">
       <widget class="QToolButton" name="cmdAddUrPerson">
        <property name="toolTip">
         <string>Добавить</string>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="nameText_3">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Контрагент:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QComboBox" name="cboPartner"/>
      </item>
      <item row="4" column="3">
       <widget class="QToolButton" name="cmdAddPartner">
        <property name="toolTip">
         <string>Добавить</string>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="nameText_9">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Статья:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="2">
       <widget class="QComboBox" name="cboSection"/>
      </item>
      <item row="5" column="3">
       <widget class="QToolButton" name="cmdAddSection">
        <property name="toolTip">
         <string>Добавить</string>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="nameText_8">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Проект:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="2">
       <widget class="QComboBox" name="cboProject"/>
      </item>
      <item row="6" column="3">
       <widget class="QToolButton" name="cmdAddProject">
        <property name="toolTip">
         <string>Добавить</string>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="nameText_7">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Сумма:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1" colspan="2">
       <widget class="QDoubleSpinBox" name="dValue">
        <property name="minimum">
         <double>-999999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="nameText_5">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Счет:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QComboBox" name="cboAccount"/>
      </item>
      <item row="3" column="3">
       <widget class="QToolButton" name="cmdAddAccount">
        <property name="toolTip">
         <string>Добавить</string>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="pnlComment">
     <property name="title">
      <string>Комментарий (необязательное поле):</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTextEdit" name="txtComment"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cmdSave">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Внести изменения</string>
       </property>
       <property name="icon">
        <iconset resource="../plan_fakt.qrc">
         <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdCancel">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Отмена</string>
       </property>
       <property name="icon">
        <iconset resource="../plan_fakt.qrc">
         <normaloff>:/images/error.png</normaloff>:/images/error.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../plan_fakt.qrc"/>
  <include location="../plan_fakt.qrc"/>
  <include location="../plan_fakt.qrc"/>
  <include location="../plan_fakt.qrc"/>
  <include location="../plan_fakt.qrc"/>
  <include location="../plan_fakt.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cmdCancel</sender>
   <signal>clicked()</signal>
   <receiver>ZOperationsForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>423</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
